backend/tests/test_git_ops.py

```
@@ -154,37 +161,49 @@ def test_error_for_missing_repo(self, mock_exists):
-                assert result["binary"] == True
-                assert result["content"] is None
+        # Mock resolve to return a path that passes the security check
+        mock_resolved = MagicMock(spec=Path)
+        mock_resolved.is_relative_to.return_value = True
+        mock_resolved.suffix = ".png"
+        mock_resolved.stat.return_value.st_size = 1024
+        mock_resolve.return_value = mock_resolved
+        
+        result = read_file("user", "repo", "image.png")
+        assert result["status"] == "success"
+        assert result["binary"] == True
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `result["binary"]:` for truth checks

Replace with `result["binary"]`

(E712)